File: clients/typescript-angular/API/Client/Transaction.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface Transaction {

        transactID: string;

        account?: number;

        currency?: string;

        transactType?: string;

        amount?: number;

        fee?: number;

        transactStatus?: string;

        address?: string;

        tx?: string;

        text?: string;

        transactTime?: date;

        timestamp?: date;
    }

}

File: clients/scala/src/main/scala/io/swagger/client/model/Transaction.scala
package io.swagger.client.model

import java.util.Date



case class Transaction (
  transactID: String,
  account: Number,
  currency: String,
  transactType: String,
  amount: Number,
  fee: Number,
  transactStatus: String,
  address: String,
  tx: String,
  text: String,
  transactTime: Date,
  timestamp: Date)
  
